src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -16,37 +18,53 @@
         ┗╸ Placeholders: %players_online%, %max_players_online%, %local_players_online%, %current_date%, %current_time%, %username%, %server%, %ping%, %prefix%, %suffix%, %role%""")
 public class Settings {
 
-    @YamlKey("header")
-    private String header = "&rainbow&Running Velocitab v" + BuildConstants.VERSION + " by William278";
-    @YamlKey("footer")
-    private String footer = "[There are currently %players_online%/%max_players_online% players online](gray)";
-    @YamlKey("format")
-    private String format = "&7[%server%] &f%prefix%%username%";
-    @YamlKey("excluded_servers")
-    private ArrayList<String> excludedServers = new ArrayList<>();
-    @YamlKey(("update_rate"))
+    @YamlKey("headers")
+    private Map<String, String> headers = Map.of("default", "&rainbow&Running Velocitab v" + BuildConstants.VERSION + " by William278");
+    @YamlKey("footers")
+    private Map<String, String> footers = Map.of("default", "[There are currently %players_online%/%max_players_online% players online](gray)");
+    @YamlKey("formats")
+    private Map<String, String> formats = Map.of("default", "&7[%server%] &f%prefix%%username%");
+    @YamlKey("server_groups")
+    private Map<String, List<String>> serverGroups = Map.of("default", List.of("lobby1", "lobby2", "lobby3"));
```

Comment:
maybe a @YamlComment here with a very short explanation of what this is?

Reply:
good point. i'll add a fallback group and YamlComment